com.unity.render-pipelines.high-definition/Documentation~/HDRP-Features.md

```diff
@@ -272,18 +302,52 @@ HDRP includes its own purpose-built implementation for post-processing to produc
 
 ![Posterize](Images/HDRPFeatures-CustomPostProcess.png)
 
-HDRP allows you to add your own custom post processes integrated with the volume framework. They can be injected after opaque and sky object, before builtin post processes or after builtin post processes. For more information, see the [Custom Post-processing documentation](Custom-Post-Process.md).
+HDRP allows you to add your own custom post processes integrated with the volume framework. They can be injected after opaque and sky object, before the [temporal anti-aliasing](Anti-Aliasing.md#TAA) pass, before builtin post processes or after builtin post processes. For more information, see the [Custom Post-processing documentation](Custom-Post-Process.md).
+
+### Path tracing
+
+#### Path-traced depth of field
+
+![](Images/Path-traced-DOF-Feature.png)
```

Comment:
Strangely, if I go to View file in the top right, this image or the one below (  ![](Images/Path_tracing_recording-Feature.png)  ) aren't showing up. I can't immediately see what's up but it might be a git sync thing. If you can see them in Typora (or whatever you're using) though then it's fine :)

Reply:
Checked this with you and it does appear to be fine :)